addon/mixins/prop-types.js

```diff
@@ -37,7 +39,9 @@ const helpers = {
       return
     }
 
-    _.forIn(propTypes, (def, name) => {
+    Object.keys(propTypes).forEach(name => {
```

Comment:
Made a pass at writing this using `for of` and destructuring - couldn't wrap my brain around it and fell back to an easy implementation.  If anyone wants to take a crack at that for my sanity...👍